src/frontend/src/icp/components/fee/EthereumFeeContext.svelte

```diff
@@ -16,18 +15,19 @@
 	} from '$icp/stores/ethereum-fee.store';
 	import { token } from '$lib/stores/token.store';
 	import { isTokenIcrcTestnet } from '$icp/utils/icrc-ledger.utils';
-	import { tokenAsIcToken, tokenWithFallbackAsIcToken } from '$icp/derived/ic-token.derived';
+	import { tokenAsIcToken } from '$icp/derived/ic-token.derived';
+	import {
+		isConvertCkErc20ToErc20,
+		isConvertCkEthToEth
+	} from '$icp-eth/utils/cketh-transactions.utils';
 
 	export let networkId: NetworkId | undefined = undefined;
 
-	let ckETH = false;
-	$: ckETH = isTokenCkEthLedger($tokenWithFallbackAsIcToken);
-
-	let ckErc20 = false;
-	$: ckErc20 = isTokenCkErc20Ledger($tokenWithFallbackAsIcToken);
+	let ckEthConvert = false;
+	$: ckEthConvert = isConvertCkEthToEth({ token: $tokenAsIcToken, networkId });
```

Comment:
The logic is changing here no, are you sure the `networkId` passed as property is the same as the one derived in `$tokenWithFallbackAsIcToken`? Wouldn't it be more safe to stick to `$tokenWithFallbackAsIcToken`?

Reply:
in this specific case, networkId is given as prop to the component:

	`export let networkId: NetworkId | undefined = undefined;`
	
I wanted to change from `tokenWithFallbackAsIcToken` to `tokenAsIcToken` because the only difference between the two of them is that the first one is never `undefined` (that is not needed anymore with the new check-function).

But I agree that it could lead to change in logic, so I can change it back if you want

Reply:
Yes, please do.

Reply:
done